docs/install/create-project-cookieplone.md

```diff
@@ -0,0 +1,315 @@
+---
+myst:
+  html_meta:
+    "description": "Create a Plone project with the Volto frontend (Volto 18+)"
+    "property=og:description": "Create a Plone project with the Volto frontend (Volto 18+)"
+    "property=og:title": "Create a Plone project with the Volto frontend (Volto 18+)"
+    "keywords": "Plone, Plone 6, Volto, create, project, install, cookieplone"
+---
+
+
+(create-project-cookieplone-label)=
+
+# Create a project with the Volto frontend (Volto 18+)
+
+This chapter describes how you can create a web application project using Plone, with full control over development and deployment.
+
+If instead you want to contribute to a Plone package, see {doc}`/contributing/index`.
+
+:::{note}
+These instructions are valid for unstable versions of Volto, version 18.0.0-alpha.43 and above.
+To create a project using a stable Plone release, see {doc}`/install/create-project`.
+:::
+
+:::{note}
+These instructions create a project using the Volto frontend.
+If you'd like to use the Classic UI instead, see {doc}`/install/create-project-classicui`.
```

Comment:
```suggestion
If you'd like to use the Classic UI instead, see {doc}`/install/create-project-classic-ui`.
```

Reply:
I rewrote the introduction, so this is no longer needed.